deployment/ansible/roles/model-my-watershed.app/templates/systemd-app.service.j2

```diff
@@ -0,0 +1,26 @@
+[Unit]
```

Comment:
This uses a template from Gunicorn for how to manage Gunicorn processes with `systemd`.